requirements.txt

```diff
@@ -1,6 +1,6 @@
+-e git+https://opendev.org/jjb/python-jenkins.git@570a143c74d092efaf9bc68a3bae9b15804f4a63#egg=python-jenkins
 pyyaml==5.4.1
 jinja2==2.11.3
-python-jenkins==1.7.0
```

Comment:
WRT python-jenkins - I'd really rather not have the project be using whatever is on top of master, this is messy. python-jenkins is an OpenStack project - why not reach out ot the community and ask them to draft a new release?

Reply:
I tried to contact of one the contributors. I'll update when I have an answer

Reply:
Any word @itzikb?

Reply:
@itzikb  made a great job and we new pre-release in pypi https://pypi.org/project/python-jenkins/1.8.0.0a0/

Reply:
Fantastic news!

Curious, why did they issue it as a "prerelease" instead of simply "1.8.0"?

Irregardless, let's use this tag for now